packages/theme-base/sd.config.js

```diff
@@ -51,11 +51,16 @@ module.exports = {
           destination: 'dist/theme-unminified.js',
           format: 'javascript/cjs-nested',
         },
+        {
+          destination: 'dist/AmplifyTheme.ts',
```

Comment:
I'm sure this already works, but if it's a type would we export it as `dist/AmplifyTheme.d.ts` or similar?

Reply:
Nope you are right, will fix.